frontend/src/app/components/application-top-bar/application-top-bar.component.ts

```diff
@@ -57,4 +61,6 @@ export class ApplicationTopBarComponent implements OnInit, OnDestroy {
       }
     });
   }
+
+  protected readonly window = window;
```

Comment:
is this really needed?

Reply:
Yes (click) doesn't have access to it otherwise